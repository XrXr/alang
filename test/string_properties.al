main :: proc () {
	a := "this is a string"
	print_int(a.length)
	ptr := a.data
	// should be 'a'
	print_int(@(ptr + 8))
	// should be 0
	print_int(@(ptr + a.length))
}
