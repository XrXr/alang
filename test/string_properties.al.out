16
97
0
